core/primer.md

```diff
@@ -152,3 +152,15 @@ Let's walk through a complex example:
 - The result is: v8 regardless of whether v8 was created with latest=true or
   not
 
+# Potential Extensions
+
+- `createdby`, `modifiedby`<br>
```

Comment:
I'd expect a trailing double space to end up as a Markdown line break.

Reply:
hmm, will need to think about that since we have a check for extra trailing spaces :-)